Orange/widgets/unsupervised/owsilhouetteplot.py

```diff
@@ -308,6 +317,11 @@ def commit(self):
         self.send("Selected Data", selected)
         self.send("Other Data", other)
 
+    def send_report(self):
+        self.report_plot()
+        self.report_caption("Silhouette plot with ({} distance)".
```

Comment:
Why is the second part of this sentence in parentheses?
I think either the word "with" or the parentheses need to be removed.